### PR TITLE
965: Added a new context dependent Action for the acl.xml file generation

### PR DESCRIPTION
**Pull request by doninAtwix**
**Description** (*)
965: Added a new context dependent Action for the acl.xml file generation

**Fixed Issues (if relevant)**

1. Fixes magento/magento2-phpstorm-plugin#965

**Questions or comments**

**Contribution checklist** (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with integration/functional tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: src/com/magento/idea/magento2plugin/actions/context/xml/NewAclXmlAction.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import com.magento.idea.magento2plugin.util.magento.GetMagentoModuleUtil;
+import org.jetbrains.annotations.NotNull;
+
+public class NewAclXmlAction extends AbstractContextAction {
+
+    public static final String ACTION_NAME = "Magento 2 ACL File";
+    public static final String ACTION_DESCRIPTION = "Create a new Magento 2 acl.xml file";
+
+    /**
+     * New routes.xml file generation action constructor.
```
Comment (bohdan-harniuk) [inline src/com/magento/idea/magento2plugin/actions/context/xml/NewAclXmlAction.java:24]: Please, change annotation to appropriate one.

**Review by bohdan-harniuk — CHANGES_REQUESTED** [review]
Hello, @doninAtwix!

Please, check my comment.
Also, do not forget to update your branch from the mainline.

File: src/com/magento/idea/magento2plugin/actions/context/xml/NewAclXmlAction.java (hunk 1 above)
Comment (doninAtwix) [reply src/com/magento/idea/magento2plugin/actions/context/xml/NewAclXmlAction.java]: Hi @bohdan-harniuk .
Thanks for your code review. I fixed. Please check.

**Review by bohdan-harniuk — APPROVED** [review]
Thank you!

**Review by eduard13 — APPROVED** [review]

**Pull request merged**